CMakeLists.txt

```diff
@@ -55,6 +55,13 @@ else()
   set(SAVE_DECOMPS 0)
 endif()
 
+if(PIO_MAX_CACHED_IO_REGIONS)
+  message (STATUS "PIO_MAX_CACHED_IO_REGIONS is " ${PIO_MAX_CACHED_IO_REGIONS})
+else()
+  set(PIO_MAX_CACHED_IO_REGIONS 65536)
+  message (STATUS "NOT DEFINED: PIO_MAX_CACHED_IO_REGIONS, use " ${PIO_MAX_CACHED_IO_REGIONS})
```

Comment:
Minor edit : Can you change the messages to
 (when PIO_MAX_CACHED_IO_REGIONS is defined),
``` " Using PIO_MAX_CACHED_IO_REGIONS = "  ${PIO_MAX_CACHED_IO_REGIONS}```
 (when PIO_MAX_CACHED_IO_REGIONS is not defined),
``` " Using PIO_MAX_CACHED_IO_REGIONS = "  ${PIO_MAX_CACHED_IO_REGIONS} " (default) " ```